src/file_utils.cr

```diff
@@ -1,6 +1,30 @@
 module FileUtils
   extend self
 
+  # Changes the current working directory of the process to the given string *path*.
+  # Alias of Dir.cd.
+  # ```
+  # FileUtils.cd("to/directory")
+  # ```
+  def cd(path : String)
+    Dir.cd(path)
+  end
+
+  # Changes the current working firectory of the process to the given string *path*
+  # and invoked the block, restoring the original working directory when the block exits.
+  # Alias of Dir.cd with block.
+  # ```
+  # FileUtils.cd("to/directory") { puts "Do something useful here!" }
+  # ```
+  def cd(path : String, &block)
+    Dir.cd(path, &block)
```

Comment:
Please implement this as:

```cr
Dir.cd(path) { yield }
```

This avoids closures, as described [here](https://crystal-lang.org/docs/syntax_and_semantics/block_forwarding.html)  (might be improved in the future)